apps/codeforafrica/src/lib/data/seo.js

```diff
@@ -0,0 +1,61 @@
+import site from "@/codeforafrica/utils/site";
+
+function stringifyDescription(description) {
+  if (!description || !Array.isArray(description)) {
```

Comment:
Isn't 👇🏽 enough? i.e. Array.isArray handles null/undefined and description will always be an array if present, no?

```suggestion
  if (!Array.isArray(description)) {
```